README.md

```diff
@@ -13,14 +13,18 @@ You will need to get a pulsar broker address and authentication token to pass to
 #### 1. Launch the Server
 The server can be launched from anywhere with a stable network connection. You can run it from the cobalts for example. For now, set `--timeout-to-clients` and `--timeout-from-clients` to a large value like 300000. This will persist the server in case the clients don't start up right away.
 ##### Figure Your Args
+###### Environment Variables
+```
+export SKYSCAN_BROKER_AUTH=$(cat ~/skyscan-broker.token)
```

Comment:
It's a bit weird that this is the only env variable, and it's really important.  Whereas the rest are cmd args, and several of those can be ignored.

Reply:
We want to put auth tokens as env vars to obfuscate, right? I figure env vars are things that wouldn't need to change from one scan to another. 

I suppose we could move the skydriver and broker addresses, plus the timeouts, to env vars. What do you think?

Reply:
Yeah, that's a k8s requirement for obfuscation.  It might be good to make it clear why it's an env var (one-line comment or readme doc).

I'd be interested in moving the timeouts at least.

Reply:
I think holding off on moving the timeout values, for now, is best since those values are still being tweaked--but I'll do the "env var as CL arg default" pattern.

Reply:
Working on some README updates

Reply:
> I think holding off on moving the timeout values, for now, is best since those values are still being tweaked--but I'll do the "env var as CL arg default" pattern.

On second thought, I'm going to change these to env vars. I prefer having one source of truth.